### PR TITLE
Remove swap navigation action

### PR DESCRIPTION
**Pull request by Cal-L**
### **Description**

As reported by @cortisiko in RC v3.8.0

ISSUE 1
After performing a swap on a custom network (I used matic) I am logged out of the app:
http://recordit.co/NS0JnkXA0U

To reproduce:
1. Add Matic 
2. Go to swaps
3. Swap any token. 
4. Upon submitting the transaction you are logged out

### **Technical Details**

- There is an extra navigation action that triggers from a txn finished event
- We need to remove this nav action from the `Main` file

### **Acceptance Criteria**

- Same scenario in the description should be fixed

### **References**

- [Slack discussion](https://consensys.slack.com/archives/C02NZ42SJ4W/p1638839930029400)

### CODE REVIEW

File: app/components/Views/Wallet/index.tsx [hunk 1]
```diff
@@ -87,6 +95,12 @@ const Wallet = ({ navigation }: any) => {
 	 */
 	const wizardStep = useSelector((state: any) => state.wizard.step);
 
+	// Prevent Android back handler
+	useEffect(() => {
```
Comment (sethkfman) [inline app/components/Views/Wallet/index.tsx:99]: What is the purpose of preventing the back button here?

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮

File: app/components/Views/Wallet/index.tsx (hunk 1 above)
Comment (Cal-L) [reply app/components/Views/Wallet/index.tsx:99]: This is how you override Android back button behavior. Returning true in the callback prevents the event from bubbling up.

Reference: https://reactnative.dev/docs/backhandler#pattern

**Review by gantunesr — APPROVED** [review]
LGTM

File: app/components/Views/Wallet/index.tsx (hunk 1 above)
Comment (sethkfman) [reply app/components/Views/Wallet/index.tsx:99]: @Cal-L I thought we were disabling the back button in a different location/file

**Pull request merged**